pyomo/solvers/plugins/solvers/SCIPAMPL.py

```diff
@@ -288,6 +387,10 @@ def _postsolve(self):
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.optimal
+                    
+        # # WARNING
+        # infeasible='infeasible'                         # Demonstrated that the problem is infeasible
```

Comment:
I appreciate these comments being inserted, but can they please be cleaned up a bit? I.e., one comment per line?